### ISSUE TITLE
 APPLOVIN MAXADS GDPR IOS

### ISSUE THREAD

**Issue by sialifecom1234** [EMI-INDO/c3addon-bug#8]
hi, I use your APPLOVIN MAXADS plugin, everything is super, but it's not enough for iOS to show GDPR as soon as there is time, please do it

**Comment by EMI-INDO** [comment]
@sialifecom1234 

I thought I had added it in the previous release but I didn't.

Actually, simply displaying ATT (App Tracking Transparency) is already eligible to display ads.

Is ATT (App Tracking Transparency) not enough?

**Comment by sialifecom1234** [comment]
I may be wrong, but as I understand it, Google admob now needs GDPR for the
EU too

пн, 22 янв. 2024 г. в 23:53, EMI-INDO ***@***.***>:

**Comment by EMI-INDO** [comment]
@sialifecom1234 

Ok now updated
https://www.construct.net/en/game-assets/addons/applovin-max-ads-1847

Plugin release v1.0.2.9

- Applovin MAX consent (support IOS|Android)
- Applovin MAX consent admob mediation SDK UMP (support IOS|Android)
- New example file c3p


- Fix bug IOS
IOS export notes
SDK KEY was previously added manually in xcode now it is automatic when exported from construct 3.

**Comment by sialifecom1234** [comment]
hi, can you update the plugin? Applovin max IOS there is a serious update released for iOS

**Comment by EMI-INDO** [comment]
@sialifecom1234 

your report does not provide any documentation, 
I'm not a .applovin user, 
I'm just a plugin creator, 
you just said changes, 
what those changes are I don't know.
at least you provide a link or screenshot about your problem.

could you please clarify this issue.

**Comment by sialifecom1234** [comment]
https://dash.applovin.com/documentation/mediation/ios/getting-started/migration123

пт, 22 мар. 2024 г. в 22:47, EMI-INDO ***@***.***>:

**Comment by EMI-INDO** [comment]
@sialifecom1234 

Yes there is a slight change, but it is very important, in this section.

- Warning: If your Info.plist contains an entry for "AppLovinSDKKey", you must remove it.

Here I have to update 2 plugins, 
first is cordova plugin, second is c3 plugin,
Please wait for the next 2 to 3 days I will update the plugin.

**Comment by sialifecom1234** [comment]
Okay, thanks for the update. You are very kind

сб, 23 мар. 2024 г. в 00:45, EMI-INDO ***@***.***>:

**Comment by sialifecom1234** [comment]
Hello. Not just a couple of days, but a couple of months have passed. Perhaps you abandoned the plugin?
